backend/.env.example

```diff
@@ -17,3 +17,8 @@ FLASK_ENV=development
 
 # Flask monitoring dashboard
 DASHBOARD_CONFIG=fmd_config.cfg
+
+# Cellsam server
+CELLSAM_IP=127.0.0.1
+CELLSAM_PORT=8765
+CELLSAM_SERVER_VERSION=
```

Comment:
@rdilip for now, let's record the commit hash of the cellsam compute server that works with this version of deepcell-label. In principle we can use tags as well, which may make more sense once the formatting of messages etc. is completely decided.

The most elegant way to do this would be to include the cellsam server as a submodule in this repo; however, mixing public and private repos in this way is likely to cause problems (I don't even know if/how it's supported). Nevertheless, something to keep in mind if the cellsam code is released in the future.